test/functional/services/common/png.ts

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { dirname } from 'path';
+import { promisify } from 'util';
+import { promises as fs, writeFile, readFileSync, mkdir } from 'fs';
+import path from 'path';
+import { comparePngs, PngDescriptor } from '../lib/compare_pngs';
+import { FtrProviderContext, FtrService } from '../../ftr_provider_context';
+
+const mkdirAsync = promisify(mkdir);
+const writeFileAsync = promisify(writeFile);
```

Comment:
nit: I guess we could just use `fs.mkdir` and `fs.writeFile` and remove the `util` import?

Reply:
👍